### ISSUE TITLE
PyPI package

### ISSUE THREAD

**Issue by DavidLP** [SiLab-Bonn/pyLandau#2]
It was reported that the pyLandau package hosted on PyPI does not work with python 3.5 under Ubuntu 12. The fit example fails with the following error:
```
Traceback (most recent call last):
  File "fit.py", line 4, in <module>
    from pyLandau import landau
ImportError: dynamic module does not define module export function (PyInit_landau)
```

Packaging errors like this are not easy to spot, since there is to my knowledge no easy way to check this with Travis CI. Most likely [this issue](http://stackoverflow.com/questions/8024805/cython-compiled-c-extension-importerror-dynamic-module-does-not-define-init-fu) is a fix. I will try it in the next release.

**Comment by DavidLP** [comment]
So far you can always install pyLandau with
```
pip install https://github.com/SiLab-Bonn/pyLandau/archive/master.zip
```

**Comment by DavidLP** [comment]
I just installed the package under Xubunutu 14.04 LTS and Miniconda2 Python with
```
pip install pylandau
```

No error occurred.

**Comment by DavidLP** [comment]
Fixed in 2.0.0
